code/WorkInProgress/rewardsLocker.dm

```diff
@@ -283,6 +283,41 @@
 		new /obj/item/pod/paintjob/tronthing(get_turf(activator))
 		return 1
 
+/datum/achievementReward/respirator
+	title = "(Skin) Gas Respirator"
+	desc = "Replaces a gas mask you're wearing with a gas respirator. Use inhand to change style."
+	required_medal = "Old Enemy"
+	once_per_round = FALSE
+	var/sacrifice_path = /obj/item/clothing/mask/gas
+	var/reward_path = /obj/item/clothing/mask/gas/respirator
+
+	rewardActivate(var/mob/activator)
+		if (!istype(activator))
+			return
+
+		if (activator.wear_mask && istype(activator.wear_mask, /obj/item/clothing/mask/gas))
+			var/O = locate(sacrifice_path) in activator.contents
+			var/found = FALSE
+			
+			if (istype(O, sacrifice_path))
+				var/obj/item/clothing/mask/gas/E = O
+				activator.remove_item(E)
+				found = TRUE
+				qdel(E)
+
+			if (!found)
+				boutput(activator, "You must be wearing a gas mask to claim this reward.")
+
+			var/obj/item/clothing/mask/gas/respirator/GR = new reward_path()
+			if (!istype(GR))
+				boutput(activator, "Something went wrong. The reward path got screwed up somehow. Call 1-800-CODER for gear modification support.")
+
+			GR.set_loc(get_turf(activator))
+			activator.put_in_hand(GR)
+			activator.u_equip(GR)
+			boutput(activator, "You replace some components of the gas mask, refitting it into a [GR]!")
+		return
+
```

Comment:
Also shouldn't skins just reskin the original item, not give you a new one? (Additionally, you forget to unequip and delete the old one)